cmd/test.go

```diff
@@ -21,7 +21,11 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	var testCmd = &cobra.Command{
 		Use:   "test",
 		Short: "Test your project in the local Appsody environment",
-		Long:  `This starts a docker container for your project and runs your test in it.`,
+		Long: `Test your project in the local Appsody environment, starting a container for your project, running the stack provided and user specified tests.
+		
+You must be in the base directory of your Appsody project when running this command.`,
```

Comment:
I think we have said: "Run this command from the root directory of your Appsody project" in other places